### ISSUE TITLE
Could not translate Error: Request failed with status code 429 + Cannot find module '@typescript-eslint/parser'

### ISSUE THREAD

**Issue by mbecker** [ioBroker/create-adapter#482]
Hi,

the issue is the same for Windows 10 nodejs v13.9.0 and Linux nodejs v10.18.0 (Raspberry Pi). The error is for "Windows Admin", "Windows Normal User", "Linux Sudo User" and "Linux Normal User".

The following is the issue:

```sh
pi@pi:/opt/iobroker/node_modules/iobroker.natsclient $ sudo npx @iobroker/create-adapter
npx: installed 173 in 60.269s

=====================================================
   Welcome to the ioBroker adapter creator v1.24.0!
=====================================================

You can cancel at any point by pressing Ctrl+C.

Let's get started with a few questions about your project!
✔ Please enter the name of your project: · testpub
✔ Which title should be shown in the admin UI? · test publisher
✔ Please enter a short description: · test publisher (client)
✔ Enter some keywords (separated by commas) to describe your project: · test,streaming,messaging,broker
✔ If you have any contributors, please enter their names (seperated by commas): · email@email.com

Nice! Let's get technical...
✔ How detailed do you want to configure your project? · no
✔ Which features should your project contain? · adapter
✔ Which category does your adapter fall into? · protocols
✔ From where will the adapter get its data? · local
✔ How will the adapter receive its data? · poll
✔ Which language do you want to use to code the adapter? · JavaScript
✔ Which of the following tools do you want to use? · ESLint, type checking
✔ Do you prefer tab or space indentation? · Tab
✔ Do you prefer double or single quotes? · double

Almost done! Just a few administrative details...
✔ Please enter your name (or nickname): · mbecker
✔ What's your name/org on GitHub? · mbecker
✔ What's your email address? · email@email.com
✔ Which license should be used for your project? · MIT License

That's it. Please wait a minute while I get this working...
[1/2] Generating files...
Could not translate to "ru": Error: Request failed with status code 429

Could not translate to "zh-cn": Error: Request failed with status code 429

Could not translate to "it": Error: Request failed with status code 429

Could not translate to "pt": Error: Request failed with status code 429

Could not translate to "pl": Error: Request failed with status code 429

Could not translate to "de": Error: Request failed with status code 429

Could not translate to "es": Error: Request failed with status code 429

Could not translate to "de": Error: Request failed with status code 429

Could not translate to "fr": Error: Request failed with status code 429

Could not translate to "nl": Error: Request failed with status code 429

Could not translate to "nl": Error: Request failed with status code 429

Could not translate to "ru": Error: Request failed with status code 429

Could not translate to "de": Error: Request failed with status code 429

Could not translate to "ru": Error: Request failed with status code 429

Could not translate to "it": Error: Request failed with status code 429

Could not translate to "pl": Error: Request failed with status code 429

Could not translate to "pt": Error: Request failed with status code 429

Could not translate to "fr": Error: Request failed with status code 429

Could not translate to "zh-cn": Error: Request failed with status code 429

Could not translate to "es": Error: Request failed with status code 429

Could not translate to "pt": Error: Request failed with status code 429

Could not translate to "nl": Error: Request failed with status code 429

Could not translate to "fr": Error: Request failed with status code 429

Could not translate to "it": Error: Request failed with status code 429

Could not translate to "es": Error: Request failed with status code 429

Could not translate to "pl": Error: Request failed with status code 429

Could not translate to "zh-cn": Error: Request failed with status code 429

Could not translate to "de": Error: Request failed with status code 429

Could not translate to "ru": Error: Request failed with status code 429

Could not translate to "pt": Error: Request failed with status code 429

Could not translate to "nl": Error: Request failed with status code 429

Could not translate to "fr": Error: Request failed with status code 429

Could not translate to "it": Error: Request failed with status code 429

Could not translate to "es": Error: Request failed with status code 429

Could not translate to "pl": Error: Request failed with status code 429

Could not translate to "zh-cn": Error: Request failed with status code 429

(node:6578) UnhandledPromiseRejectionWarning: Error: Cannot find module '@typescript-eslint/parser'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.tsFixQuotes (/root/.npm/_npx/6578/lib/node_modules/@iobroker/create-adapter/build/src/lib/tools.js:260:54)
    at files.map (/root/.npm/_npx/6578/lib/node_modules/@iobroker/create-adapter/build/src/lib/createAdapter.js:87:41)
    at Array.map (<anonymous>)
    at formatFiles (/root/.npm/_npx/6578/lib/node_modules/@iobroker/create-adapter/build/src/lib/createAdapter.js:70:18)
    at Object.createFiles (/root/.npm/_npx/6578/lib/node_modules/@iobroker/create-adapter/build/src/lib/createAdapter.js:52:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:6578) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 38)
(node:6578) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

P.s. I'm trying to create the adapter in the directory "node_modules" from iobroker to start the development process with Visual Studio SSH Client. With Windows I'm trying to use the adapter on my local PC (same error as stated above).

**Comment by DutchmanNL** [comment]
confirmed also on MAC OSx

**Comment by AlCalzone** [comment]
The 2nd part of the error ( Cannot find module '@typescript-eslint/parser') is fixed in v1.24.1. The 1st one seems to be a rate-limit for google translate and should go away after an hour or so. The files should still be there, but they are not auto-translated. I don't think I can do anything about that, except logging less.
